Ex2.py

```diff
@@ -0,0 +1,9 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# *Пример:*
+
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0)
+
+n = int(input('Введите число '))
+x = (n // 100 + (n // 10) % 10 + n % 10)
```

Comment:
Все верно, Ольга, однако скобки тут вообще нигде не нужны.
```suggestion
x = n // 100 + n // 10 % 10 + n % 10
```